src/utils/parser/utils.js

```diff
@@ -102,6 +102,14 @@ function getFunctionName(path) {
     return parent.id.name;
   }
 
+  if (parent.right && parent.right.type == "ArrowFunctionExpression") {
```

Comment:
You might be able to avoid the two conditions by using the `isFunction` helper I wrote.
```js
if (parent.right && isFunction(parent.right) {
  return parent.left.property.name;
}
```